### PR TITLE
Security service initialization

### LINKED ISSUE #1: Initialize security service

### PR DESCRIPTION
**Pull request by Asmanser**

### CODE REVIEW

File: security-service/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation("org.springframework.boot:spring-boot-starter")
+    implementation("org.springframework.boot:spring-boot-starter-web")
+    implementation("org.jetbrains.kotlin:kotlin-reflect")
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+    compileOnly("org.projectlombok:lombok")
```
Comment (AndreiMayorau) [inline security-service/build.gradle.kts]: Do we really need lombok here?

File: security-service/HELP.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# SECURITY SERVICE
```
Comment (AndreiMayorau) [inline security-service/HELP.md]: Pls, remove autogenerated HELP file.

**Review by KoizumiHana — APPROVED** [review]

**Review by KoizumiHana — APPROVED** [review]

**Review by KoizumiHana — COMMENTED** [review]
Pls, resolve comments. >_<

**Comment by AndreiMayorau** [comment]
LGTM!

**Pull request merged**